sourcemod/gamedata/open-fortress.txt

```diff
@@ -75,14 +75,14 @@
 			{
 				"library"	"server"
 				// AddCond and RemoveCond are made almost the exact same way so oh well. Probably won't break that often
-				"windows"	"\x55\x8B\xEC\x53\x56\x8B\x75\x08\x8B\xD9\x89\x75\x08\x57\x81\xFE\x80\x00\x00\x00\x7D\x2A\x83\xFE\x60\x7D\x2A\x83\xFE\x40\x7D\x2A\x83\xFE\x20\x7D\x2A\x8D\x7B\x1C\x8B\x47\xE4\x8D\x4F\xE4\x57\xFF\x10\xEB\x2A\x8D\x7B\x20\x8B\x47\xE0\x8D\x4F\xE0\x57\xFF\x10\x8D\x46\xE0\xEB\x2A\x8D\x7B\x24\x8B\x47\xDC\x8D\x4F\xDC\x57\xFF\x10\x8D\x46\xC0\xEB\x2A\x8D\x7B\x28\x8B\x47\xD8\x8D\x4F\xD8\x57\xFF\x10\x8D\x46\xA0\xEB\x2A\x8D\x7B\x2C\x8B\x47\xD4\x8D\x4F\xD4\x57\xFF\x10\x8D\x46\x80\x89\x45\x08\x8B\x4D\x08\xB8\x01\x00\x00\x00\xD3\xE0\x8D\x4D\x0C"
+				"windows"	"\x55\x8B\xEC\x51\x8B\x81\x24\x03\x00\x00"
 				"linux"		"\x55\x89\xE5\x57\x56\x53\x83\xEC\x1C\x8B\x75\x0C\x8B\x5D\x08\x83\xFE\x7F"
 				"mac"		"\x55\x89\xE5\x57\x56\x53\x83\xEC\x1C\x8B\x75\x0C\x8B\x5D\x08\x83\xFE\x7F"
 			}
 			"RemoveCondition"
 			{
 				"library"	"server"
-				"windows"	"\x55\x8B\xEC\x53\x56\x8B\x75\x08\x8B\xD9\x89\x75\x08\x57\x81\xFE\x80\x00\x00\x00\x7D\x2A\x83\xFE\x60\x7D\x2A\x83\xFE\x40\x7D\x2A\x83\xFE\x20\x7D\x2A\x8D\x7B\x1C\x8B\x47\xE4\x8D\x4F\xE4\x57\xFF\x10\xEB\x2A\x8D\x7B\x20\x8B\x47\xE0\x8D\x4F\xE0\x57\xFF\x10\x8D\x46\xE0\xEB\x2A\x8D\x7B\x24\x8B\x47\xDC\x8D\x4F\xDC\x57\xFF\x10\x8D\x46\xC0\xEB\x2A\x8D\x7B\x28\x8B\x47\xD8\x8D\x4F\xD8\x57\xFF\x10\x8D\x46\xA0\xEB\x2A\x8D\x7B\x2C\x8B\x47\xD4\x8D\x4F\xD4\x57\xFF\x10\x8D\x46\x80\x89\x45\x08\x8B\x4D\x08\xB8\x01\x00\x00\x00\xD3\xE0\x8D\x4D\x08"
+				"windows"	"\x55\x8B\xEC\x51\x8B\x81\x24\x03\x00\x00"
```

Comment:
I'm quite positive that the original AddCond/RemoveCond signatures are correct. Checked IDA just now and I don't see any functions that have these signatures.

I'm taking your word for it with the remaining ones, as they seem valid.